assets/base.css

```diff
@@ -2326,7 +2326,7 @@ product-info .loading__spinner:not(.hidden) ~ *,
 
 /* section-header */
 .section-header.shopify-section-group-header-group {
-  z-index: 3;
+  z-index: 4;
```

Comment:
Have we tested the entire theme to ensure this doesnt break anything? 👀

Reply:
I dont have any specific scenarios. I would test it with cart notification, search popup etc to ensure it doesn't break that experience. Since this changes the header globally we want to ensure it doesnt affect any existing features